templates/home.html

```diff
@@ -5,7 +5,7 @@
   <title>SayLess - Home</title>
   <!-- Bootstrap CSS and fa-icon -->
   <link rel="stylesheet" href="/include/bootstrap.css">
-  <link rel="stylesheet" href="LOCAL homepage.css">
+  <link rel="stylesheet" href="/statics/home.css">

```

Comment:
No need to have statics. Keep it as /home.css

Reply:
Done